app/routes.js

```diff
@@ -64,14 +64,43 @@ module.exports = {
     method: 'get',
     handler: require('./components/Activity.jsx'),
     action: function (context, payload, done) {
-      var activityId  = payload.get('params').get('id');
+      var activityId  = payload.get('params').get('id')+"";
       context.dispatch('UPDATE_PAGE_TITLE', { pageTitle: activityId + ' Aktywnosc' });
       context.executeAction(actions.showActivity, { id: activityId }, function() {
         done();
       })
+      context.executeAction(actions.loadVolonteers, {}, function() {
+        done();
+      })
```

Comment:
Tak jak pisałem wyżej, nie chcemy tu tak naprawdę ładować listy wszystkich użytkowników w systemie, więc tej akcji też tu nie chcemy, ale napiszę tylko na przyszłość, że gdybyśmy chcieli coś takiego, to funkcja zwrotna (callback) `done` może być wywołana tylko raz, więc powyższy zapis jest błędny. Powinno być coś takiego:

```
context.executeAction(actions.showActivity, { id: activityId }, function() {
  context.executeAction(actions.loadVolonteers, {}, function() {
    done();
  })
})
```